components/webdriver_server/lib.rs

```diff
@@ -146,9 +168,12 @@ pub struct WebDriverSession {
     unhandled_prompt_behavior: String,
 
     // https://w3c.github.io/webdriver/#dfn-input-state-table
-    input_state_table: HashMap<String, InputSourceState>,
+    input_state_table: RefCell<HashMap<String, InputSourceState>>,
+
     // https://w3c.github.io/webdriver/#dfn-input-cancel-list
-    input_cancel_list: Vec<ActionSequence>,
+    input_cancel_list: RefCell<Vec<ActionSequence>>,
+    // https://w3c.github.io/webdriver/#dfn-actions-queue
+    // Currently, we don't need a queue because only 1 thread run actions at a time.
```

Comment:
I would omit this comment or make it part of one of the previous members rustdoc comments. It's a bit weird to have a floating comment in a struct definition and I worry that it will, over time, get separated from the members that contextualize it.